UnitsNet/UnitMath.cs

```diff
@@ -47,6 +58,16 @@ public static TQuantity Sum<TQuantity>(this IEnumerable<TQuantity> source, Enum
             return source.Select(selector).Sum(unitType);
         }
 
+        /// <summary>Returns the smaller of two <typeparamref name="TQuantity" /> values.</summary>
+        /// <typeparam name="TQuantity">The type of quantities to compare.</typeparam>
+        /// <param name="val1">The first of two <typeparamref name="TQuantity" /> values to compare.</param>
+        /// <param name="val2">The second of two <typeparamref name="TQuantity" /> values to compare.</param>
+        /// <returns>Parameter <paramref name="val1" /> or <paramref name="val2" />, whichever is smaller.</returns>
+        public static TQuantity Min<TQuantity>(TQuantity val1, TQuantity val2) where TQuantity : IComparable, IQuantity
+        {
+            return val1.CompareTo(val2) == 1 ? val2 : val1;
```

Comment:
It was just so that in case the values are equal- it is the first one that is returned (assuming this might be called with a reference type). Unlike the other extensions, the static Min/Max functions don't actually require an actual struct to work.
This reminds me I should put a few tests with HowMuch..
Do you still think I should invert the condition? Make it <= instead?

Reply:
If they are equal then it should hopefully not matter which one is returned, but it's fine as it is 👍